### ISSUE TITLE
Monochrome: half-dot-shift support

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#128]
**BerliOS Feature Request #5213**

**Date:** 2011-January-04 00:11
**Submitted By:** tomch
**Category:** General Functionality
**Assigned To:** mpohoreski
**Priority:** 5
# 

Date: 2011-Jan-04 00:11
Sender: tomch
Logged In: YES 
user_id=25188
Browser: Mozilla/5.0 (Windows; U; Windows NT 5.1; en-GB; rv:1.9.1.5) Gecko/20091102
Firefox/3.5.5 ( .NET CLR 3.5.30729; .NET4.0E)

From Stanimir  on cea2:

https://groups.google.com/group/comp.emulators.apple2/browse_frm/thread/c83fc117ce46d
446/37892dff3fbfcf1d?hl=en#37892dff3fbfcf1d

I remember on my Apple II clone (Pravetz 8C) using a
monochrome monitor there was a graphics display
behavior used in programs and games which allowed to
display smoother letters and shapes like (ASCII-art,
use monospace font):

   *\*      *\*   0 0100010 $22
    *\*    *\*    1 0010010 $92
     *\*  *\*     0 0010100 $14
      ***\*      1 0001100 $8C
       *\*       0 0001000 $08
       *\*       0 0001000 $08
       *\*       0 0001000 $08

(Note, two adjacent asterisks are used to visualize a
single pixel.)

Basically when the most significant bit is set the
pixels get shifted half-pixel right, instead of looking
like:

---

---

```
 **  **
 ****
   **
   **
   **
```

The Color (Half-Shift) mode doesn't really do the
half-shifting I'm after.  Are here any AppleWin
developers around to ask for providing such new video mode?
